salt/pillar/observer.sls

```diff
@@ -20,3 +13,10 @@ observer:
         port: 587
         user: null
         pass: null
+
+
+# the name of the database can be overwritten like this:
```

Comment:
this will be needed in every project that starts using `elife.postgresql-appdb`, but not elsewhere. It's good backward compatibility